package.json

```diff
@@ -35,7 +36,7 @@
 	"version": "0.112.0",
 	"publisher": "GitHub",
 	"engines": {
-		"vscode": "^1.100.0"
+		"vscode": "^1.102.0"
```

Comment:
@alexr00  FYI bumped engine version due to new api proposal (@osortega added `chatParticipantAdditions`).  This is necessary, right?

Reply:
Yep!